demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -322,6 +325,23 @@ export class DemoMeetingApp
   // this to `halt`, which allows us to stop and measure memory leaks.
   behaviorAfterLeave: 'spa' | 'reload' | 'halt' = 'reload';
 
+  videoUpstreamMetricsKeyStatsToShow: { [key: string]: string } = {
+    resolution: 'Resolution',
```

Comment:
This is a bit confusing since resolution is not a metrics we export.

Reply:
I don't get it. Do you mean we can remove the resolution?